app/models/case_contact.rb

```diff
@@ -65,6 +66,20 @@ def contact_made_chosen
     errors[:base] << "Must enter whether the contact was made." if contact_made.nil?
     !contact_made.nil?
   end
+
+  def allowed_edit?
+    today = Time.zone.now
+    return false if occurred_at.end_of_quarter < today
+    
+    true
+  end
```

Comment:
Since `occurred_at.end_of_quarter < today` will return `true` or `false`, you can just return that statement:

```
def allowed_edit?
  today = Time.zone.now
  occurred_at.end_of_quarter < today
end
```

Reply:
Though whoops, my operator sign is backwards!

```
def allowed_edit?
  today = Time.zone.now
  occurred_at.end_of_quarter > today
end
```